packages/botonic-core/src/hubtype-service.js

```diff
@@ -86,4 +86,10 @@ export class HubtypeService {
       }
     )
   }
+
+  static async getWebchatVisibility({ appId }) {
+    return axios.get(
+      `${HUBTYPE_API_URL}/v1/provider_accounts/${appId}/visibility/`
```

Comment:
Since it requires a new API, I suggest documenting the backend PR on the PR description https://github.com/metis-ai/hubtype-backend/pull/257

Reply:
added 👍